src/jsonrpc/server/pool.rs

```diff
@@ -27,11 +23,10 @@ pub async fn pool_start(
     unlock_method: Option<UnlockMethod>,
     prompt: Option<RawFd>,
 ) -> StratisResult<bool> {
-    if let (Some(fd), Some(kd)) = (prompt, key_get_desc(Arc::clone(&engine), id.clone()).await?) {
-        key_set(engine.clone(), &kd, fd).await?;
-    }
-
-    Ok(engine.start_pool(id, unlock_method).await?.is_changed())
+    Ok(engine
```

Comment:
Because of the change here, I think you will also want a change in ```src/bin/stratis-min/stratis-min.rs``` specifically around the error message ``` "--prompt and an unlock_method of clevis are mutually exclusive"```. I think that should no longer be true after this PR, because it is not true on the D-Bus side.

This test code here also illustrates the current behavior: https://github.com/stratis-storage/stratisd/pull/3635 .